orderer/common/types/channelinfo.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package types
+
+// ErrorResponse carries the error response an HTTP request.
+// This is marshaled into the body of the HTTP response.
+type ErrorResponse struct {
+	Error string `json:"error"`
+}
+
+// ChannelList carries the response to an HTTP request to List all the channels.
+// This is marshaled into the body of the HTTP response.
+type ChannelList struct {
+	Size          int                `json:"size"`          // The number of channels
```

Comment:
I think having a size here is redundant as you can count the number of channels

Reply:
I agree it is redundant from a program point of view. What I was thinking is that it would be useful for a human user that looks at the JSON object in an HTTP response object, say in a browser, or using curl, to immediately tell how many channels there are. There could be many, and counting the stuff in field `Channels` could be confusing.

If you strongly object I will get rid of it.

Reply:
No, you have a point.